### DISCUSSION TITLE
What are the rules for the usage of the API?

### DISCUSSION THREAD

**Discussion by RanvirChoudhary** [lukePeavey/quotable#122] [Questions]
I plan to use this API and was asking if there's anything i need to know before i start using it such as

- what are the rules for attribution? 
- is the API paid?
- does the API need an API key?

if there's anything i need to be careful of or need to know about before i start using this API please do tell me.

**Comment by lukePeavey** [comment]
This is an open/public API.

It does not require any authentication.  There are no rate limits or restrictions on usage.

Attribution is appreciated but not required.
